src/components/schema-tree.js

```diff
@@ -224,7 +224,7 @@ export default class SchemaTree extends LitElement {
     }
 
     // For Primitive Data types
-    const { type, readOrWriteOnly, constraint, defaultValue, example, allowedValues, pattern, schemaDescription, schemaTitle, deprecated } = JSON.parse(data);
+    const { type, format, readOrWriteOnly, constraint, defaultValue, example, allowedValues, pattern, schemaDescription, schemaTitle, deprecated } = JSON.parse(data);
```

Comment:
Any change we make in this file needs to also be made in the `schema-table.js` component as well.

Reply:
I like your idea of override `type` with the format, so we shouldn't need to pass this property in. Does that make sense?